03LessonThree/in-class-project/daily-planet.html

```diff
@@ -1,10 +1,53 @@
-<!DOCTYPE html>
-<html>
-    <head>
-        <title>The Daily Planet</title>
-        <link rel="stylesheet" href="css/daily-planet-style.css"/>
-    </head>
-    <body>
-        <!-- You code start here -->
-    </body>
-</html>
\ No newline at end of file
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>The Daily Planet</title>
+    <link rel="stylesheet" href="css/daily-planet-style.css" />
+</head>
+
+<body>
+    <!-- You code start here -->
+    <header>
```

Comment:
great job, Tu! very organized and correctly nested :+1:  one note: try to comment on your code a little more. this will help you stay compartmentalized.